Source/Rhetos.Dsl/MacroOrderRepository.cs

```diff
@@ -34,39 +36,38 @@ public class MacroOrder
     }
 
     /// <summary>
-    /// Reads from database the recommended order of macro concepts evaluation.
+    /// Reads from file the recommended order of macro concepts evaluation.
     /// The order is optimized to reduce number of iteration in macro evaluation.
     /// </summary>
     public class MacroOrderRepository : IMacroOrderRepository
     {
-        ISqlExecuter _sqlExecuter;
         ILogger _loadOrderLogger;
         ILogger _saveOrderLogger;
+        GeneratedFilesCache _generatedFilesCache;
 
-        public MacroOrderRepository(ISqlExecuter sqlExecuter, ILogProvider logProvider)
+        public MacroOrderRepository(ILogProvider logProvider, GeneratedFilesCache generatedFilesCache)
         {
-            _sqlExecuter = sqlExecuter;
+            _generatedFilesCache = generatedFilesCache;
             _loadOrderLogger = logProvider.GetLogger("MacroRepositoryLoad");
             _saveOrderLogger = logProvider.GetLogger("MacroRepositorySave");
         }
 
+        private const string MacroOrderFileName = "MacroOrder.json";
+
         /// <summary>
         /// Dictionary's Key is EvaluatorName, Value is EvaluatorOrder.
         /// </summary>
         public List<MacroOrder> Load()
         {
-            string sql = "SELECT EvaluatorName, EvaluatorOrder FROM Rhetos.MacroEvaluatorOrder";
-            var macroOrders = new List<MacroOrder>();
-            _sqlExecuter.ExecuteReader(sql, reader =>
-                {
-                    macroOrders.Add(new MacroOrder
-                    {
-                        EvaluatorName = reader.GetString(0),
-                        EvaluatorOrder = reader.GetDecimal(1)
-                    });
-                });
-            _loadOrderLogger.Trace(() => "\r\n" + ReportMacroOrders(macroOrders));
-            return macroOrders;
+            var macroOrderJson = _generatedFilesCache.LoadFromCache(MacroOrderFileName);

```

Comment:
No need to use `_generatedFilesCache`, this class has a different purpose. The file should be saved directly to `Paths.GeneratedFolder`.

Reply:
`_generatedFilesCache` field can now be removed from the class.